src/auth/oauth/test/oauth.test.ts

```diff
@@ -286,8 +288,16 @@ describe('validateAuthCallback', () => {
     /* eslint-enable @typescript-eslint/naming-convention */
 
     fetchMock.mockResponse(JSON.stringify(successResponse));
-    await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
-    session = await Context.SESSION_STORAGE.loadSession(cookies.id);
+    Cookies.prototype.set.mockImplementation(() => {
+      throw new Error('offline sessions should not depend on cookies');
+    });
+    Cookies.prototype.get.mockImplementation(() => {
+      throw new Error('offline sessions should not depend on cookies');
+    });
```

Comment:
Can you explain a little bit about what's happening here? Is this a new `Error` just for the context of this test?

Reply:
Yea it is -- changed it to use `fail` from jasmine so it is clearer what is going on.